deploy/Dockerfile

```diff
@@ -13,25 +13,34 @@ RUN export DBG="-g -Wall" && \
   make -e -C libpfm-4.10.1 && \
   make install -C libpfm-4.10.1
 
+RUN git clone -b v02.00.00.3820 https://github.com/intel/ipmctl/ && \
+    cd ipmctl && \
+    mkdir output && \
+    cd output && \
+    cmake -DRELEASE=ON -DCMAKE_INSTALL_PREFIX=/ -DCMAKE_INSTALL_LIBDIR=/usr/local/lib .. && \
+    make -j all && \
+    make install
+
 ADD . /go/src/github.com/google/cadvisor
 WORKDIR /go/src/github.com/google/cadvisor
 
 ENV GOROOT /usr/lib/go
 ENV GOPATH /go
-ENV GO_FLAGS="-tags=libpfm,netgo"
-RUN ./build/build.sh
+ENV GO_FLAGS="-tags=libpfm,netgo,libipmctl"
 
+RUN ./build/build.sh
 
 FROM alpine:3.12
 MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com stclair@google.com
 
-RUN apk --no-cache add libc6-compat device-mapper findutils zfs && \
+RUN apk --no-cache add libc6-compat device-mapper findutils zfs ndctl-dev && \
```

Comment:
`ndctl` should be enough, development files are not needed to run cAdvisor with libipmctl

Reply:
is this still correct? is `ndctl` enough or is `ndctl-dev` needed?

Reply:
Unfortunately not